translations/en.json

```diff
@@ -269,7 +269,7 @@
   "TYPE_UNITS_SUB_TEXT": "Select the convenient units to be used in the application",
   "LOGS": "Logs",
   "PROFILE": "Profile",
-  "COMPLETING": "Completing",
+  "COMPLETING": "Submiting...",
```

Comment:
You'd have to edit the other translations to say the same thing, unless you don't mind them saying a different thing from the English version.

Reply:
yeah true. i'll leave the other languages for now though and do it later